SUPPORT.md

```diff
@@ -2,4 +2,10 @@
 
 Thanks for trying out Ark! We welcome all feedback, please consider joining our mailing list: 
 
-- [Mailing List](http://j.hept.io/ark-list)
\ No newline at end of file
+- [Mailing List](http://j.hept.io/ark-list)
+
+# Ark Social
+ Thanks for trying out Ark! We welcome all to our Social Platforms: 
+- [Slack Channel](https://kubernetes.slack.com/messages/ark-dr)
+- [Google Group](https://groups.google.com/forum/#!forum/heptio-ark)
```

Comment:
This is identical to the Mailing List above (j.hept.io/ark-list resolves to the google group)

Reply:
The mailing list is same. But I've added Slack Channel link, Google Group link and Twitter link

Reply:
@nrb  #1017  was done by me as part of digitalocean hacktoberfest but it was made on release-0.9 branch . and I got a mail saying that if I'm still interested in getting this merged, it should be made against the master branch. So that I done the same in the master branch.

Reply:
@midhunbiju let's drop L10 since it's a duplicate of L5, then I think we can merge this.  Please amend the existing commit so there's only a single commit in the PR.

Reply:
@skriss  how can I amend the existing commit ? I don't know how to do that. 
can you please help me to do that ?

Reply:
Once you make the change, do `git add SUPPORT.md && git commit --amend`, then `git push -f` to force-push it.

Reply:
@midhunbiju The sample provided would be from the command line using the `git` executable. Are you using `git` directly, or are using git integration with an editor?